src/Evolution/Systems/GeneralizedHarmonic/Initialize.hpp

```diff
@@ -56,19 +56,29 @@ struct InitializeConstraints {
                     const ParallelComponent* const /*meta*/) noexcept {
     using compute_tags = tmpl::flatten<db::AddComputeTags<
         GeneralizedHarmonic::Tags::GaugeConstraintCompute<Dim, frame>,
+        // GeneralizedHarmonic::Tags::ThreeIndexConstraintCompute<Dim, frame>,
```

Comment:
Why comment out the three-index constraint? Also, why are these constraints necessary for this PR?

Reply:
They are not. I had added all constraints to the executable for (mostly) testing and validating purposes. I agree that maybe this change should be made outside of this PR. This also ties into <https://github.com/sxs-collaboration/spectre/pull/1558#discussion_r444492000>  which is only needed because of newly included constraints in the executable. 

I'll make this a separate PR, which would (a) change the dependencies of `FConstraint` and `TwoIndexConstraint` as discussed [above](https://github.com/sxs-collaboration/spectre/pull/1558#discussion_r444492000), and (b) include more constraint observers in the executable.